recipes-openxt/tss/tpm2.0-tss_git.bb

```diff
@@ -0,0 +1,82 @@
+SUMMARY = "Software stack for TPM2."
+DESCRIPTION = "tpm2.0-tss like woah."
+SECTION = "tpm"
+
+# This is a lie. The source for this project is covered by several licenses.
+# We're currently working on a way to make this clear for those consuming the
+# project. Till then I'm using 'BSD' as a place holder since the Intel license
```

Comment:
We already handle Intel licenses in the intel-microcode recipe, as well as iasl (and acms but that one is more complex).
Could be worth having a look see if we can do something similar here.